src/main/java/bio/terra/service/snapshotbuilder/query/Concept.java

```diff
@@ -0,0 +1,44 @@
+package bio.terra.service.snapshotbuilder.query;
+
+public class Concept extends TableVariable {
+
+  public static final String TABLE_NAME = "concept";
+  public static final String DOMAIN_ID = "domain_id";
+  public static final String CONCEPT_NAME = "concept_name";
+  public static final String CONCEPT_ID = "concept_id";
+  public static final String STANDARD_CONCEPT = "standard_concept";
+  public static final String CONCEPT_CODE = "concept_code";
+  private static final TablePointer tablePointer = TablePointer.fromTableName(TABLE_NAME);
```

Comment:
I think this works as there's per-object state in `TablePointer` but maybe it's safer to make a new one for each table, unless there's a reason to share them across all tables.